src/test/java/edu/harvard/iq/dataverse/export/SchemaDotOrgExporterTest.java

```diff
@@ -161,7 +162,7 @@ public void testExportDataset() throws Exception {
         assertEquals("2001-10-01/2015-11-15", json2.getJsonArray("temporalCoverage").getString(1));
         assertEquals(null, json2.getString("schemaVersion", null));
         assertEquals("Dataset", json2.getJsonObject("license").getString("@type"));
-        assertEquals("CC0", json2.getJsonObject("license").getString("text"));
+        assertEquals("CC0", json2.getJsonObject("license").getString("name"));
```

Comment:
Yeah - same issue but this test is for the Schema.org jsonld export file. I guess the question would be should we fix/improve by sending the license as a URL here (as would be helpful for the Google/page header metadata) or is there a reason to keep the export as is? I guess I'd suggest updating this export as well, but I don't know what relies on the current syntax. Not changing (or reverting to "text") would not provide as much info - all "Custom Dataset Terms" have the same name but each dataset gets a unique URL for its custom license that points to it's specific custom entries (on the terms tab.)

Reply:
https://github.com/IQSS/dataverse/issues/7349 is related. Among the issues that Google Search Console includes in their emailed complaints about the Schema.org exports of datasets in the Harvard Dataverse Repository is "Invalid object type for field "license. I got an email about this just last Tuesday. And I'd imagine admins of other Dataverse repositories on version 4.20+ are getting these emails if they're using Google Analytics.

The Dataverse guides stress that the Schema.org exports are apt to change as Google's guidelines evolve and they tinker with how it's used in their search results and in Google Dataset Search. I wouldn't think backwards compatibility should be a concern for making improvements to Dataverse repository's Schema.org exports. Although the last time I checked there were no negative consequences. For example, Google still seemed to index licenses even though the Dataverse software uses "text" instead of "name". But maybe Google has or will get stricter about this and maybe other platforms will start expecting Schema.org metadata to use "name" instead of "text".

Reply:
A third option (apart from "leave it the same" and "change it") is to have both with a note about which field is preferred ("name") and which is deprecated and may be removed some day ("text").